README.md

```diff
@@ -30,22 +30,19 @@ A Redis instance must be available on port 6379. An easy way to do that is:
 docker run --name redis -d -p 6379:6379 redis:6.0
 ```
 
-Run the app and worker:
+Run the server:
 
 ```bash
-npm run dev:worker
-
-# In another console
-npm run dev:app
```

Comment:
@apolkingg8 I think some of the README is still out of date, I didn't check everything.